### PR TITLE
Prefabs with Partials documentation

### PR DESCRIPTION
**Pull request by swethakundaram**

### CODE REVIEW

File: learn/app-development/custom-widgets/prefab-with-partials.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+You can import [template bundles](/learn/app-development/ui-design/page-concepts/creating-template-bundles) of Partials and Widget Templates into a Prefab Project. Choose the **Platform Type** when using the **Import Template** option to import for web and mobile projects separately.
+
+## Prefab Properties inside Partials
+
+The Main page already has access to the Prefab properties. Let us see how Partials can access these properties.
+
+### Design Tab
+
+The following Prefab has two properties, Prop1 and Prop2.
```
Comment (swethakundaram) [inline learn/app-development/custom-widgets/prefab-with-partials.md:30]: @nikhilesh-kv, can you add another point on how you get that design tab dialog?

File: learn/app-development/custom-widgets/prefab-with-partials.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+![PrefabPropertiesDialog](/learn/assets/prefabpropertiesdialog.png)
+
+These properties are available to be used in the Partials created inside the Prefab, as shown below.
```
Comment (swethakundaram) [inline learn/app-development/custom-widgets/prefab-with-partials.md:34]: @nikhilesh-kv, can you include all the properties here in this section, please?

**Comment by swethakundaram** [comment]
@prasanth-reddy-wavemaker-com and @nikhilesh-kv, let me know your comments. The document is ready to publish.

File: learn/app-development/custom-widgets/prefab-with-partials.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+2. **Widget Templates**: Create [Widget Templates](/learn/how-tos/custom-widget-template) with widgets, including Search, Radioset, Checkboxset to customize the look and feel.
+
+![PartialTypes](/learn/assets/partialTypes.png)
+
+You can import [template bundles](/learn/app-development/ui-design/page-concepts/creating-template-bundles) of Partials and Widget Templates into a Prefab Project. Choose the **Platform Type** when using the **Import Template** option to import for web and mobile projects separately.
+
+## Prefab Properties in Partial
+
+Access the general properties from the Main page by default. To access properties from a Partial, you add them from the **Config Prefab** section. Follow the steps below to achieve this.
+
```
Comment (nikhilesh-kv) [inline learn/app-development/custom-widgets/prefab-with-partials.md:27]: This gives an impression that you can add prefab properties only from partials. Can we say something like "Similar to how we can access Prefab properties from Main page, you can access them from Partials as well. To create a property, you need to go the Config Prefab section as shown below."

File: learn/how-tos/create-prefab-with-partials.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+---
+---
+In this document, the following two use cases explain how to use [Partials in a Prefab](/learn/app-development/custom-widgets/prefab-with-partials).
+
+1. [Partials in Web and Mobile](#partials-in-web-and-mobile)
+2. [Using Partial for Data Table with Row Expansion](#data-table-with-row-expansion)
+
+## Partials in Web and Mobile
+
+To build a Prefab for the web and mobile, you develop different partials for different viewports, and it segregates the code automatically.
```
Comment (nikhilesh-kv) [inline learn/how-tos/create-prefab-with-partials.md:14]: Doesn't "segregates the code automatically" sound like it automatically handles the show and hide conditions for different viewports ? "You develop different partials for different viewports, which helps in segregating the code." ?

Comment (swethakundaram) [reply learn/how-tos/create-prefab-with-partials.md:14]: Well pointed!

**Review by nikhilesh-kv — COMMENTED** [review]
Looks good to me.

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

**Pull request merged**